packages/core/src/i18n/i18n.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import merge from 'lodash.merge';
+import {brandingText} from 'src/api/branding-text';
+import {AuthClient} from 'src/auth-client';
+import {BrandingTextAPIResponse} from 'src/models/branding-text-api-response';
+import {GetLocalization, TextObject} from './screens/model';
+
+/**
+ * Fetch and merge branding properties.
+ *
+ * @param {BrandingProps} props - Branding properties.
+ * @returns {Promise<Customization>} A promise that resolves with the merged branding properties.
+ */
+const getLocalization = async (props: GetLocalization): Promise<TextObject> => {
+  const {componentCustomization, locale, providerCustomization, screen} = props;
+  /**
+   * Default stored branding
+   */
+  const module: TextObject = await import(`./screens/${screen}/${locale}.ts`); // PRIORITY 04
+
+  /**
+   * Text from console branding
+   */
+  let textFromConsoleBranding: BrandingTextAPIResponse; // PRIORITY 03
```

Comment:
Let's improve this comment and move to another line? Check the other places also.

Reply:
Improved with d78cd70a9acfc828531b7a87044630953a977341